scraper/src/documentation_spider.py

```diff
@@ -27,27 +30,51 @@ class DocumentationSpider(CrawlSpider, SitemapSpider):
     strategy = None
     js_render = False
     js_wait = 0
+    match_capture_any_scheme = r"^(https?)(.*)"
+    backreference_any_scheme = r"^https?\2(.*)$"
+    # Could be any url prefix such as http://www or http://
+    every_schemes = ["http", "https"]
+
+    @staticmethod
+    def to_any_scheme(url):
+        """Return an regex that represent the URL and is scheme agnostic """
+        return url if not re.match(DocumentationSpider.match_capture_any_scheme, url) else re.sub(
+            DocumentationSpider.match_capture_any_scheme, DocumentationSpider.backreference_any_scheme, url)
+
+    @staticmethod
+    def to_other_scheme(url):
```

Comment:
I am not sure this one works like you expect it to.

Right now if you have an https request I think it add http and also https because both match the regex

Reply:
I would still go for a non regex version of this one. In the end we can always switch back to regex if weed need.
but for two cases like this, it might be a bit too much